prdoc/pr_4586.prdoc

```diff
@@ -0,0 +1,16 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Removed `pallet::getter` usage from pallet-identity
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removed the `pallet::getter`s from `pallet-identity`.
+      The syntax `StorageItem::<T, I>::get()` should be used instead.
+
+crates:
+  - name: pallet-identity
+    bump: major
+  - name: kitchensink-runtime
+    bump: minor
```

Comment:
```suggestion
    bump: patch
```
Nothing was added, or?

Reply:
Is polkadot-sdk semver typically:
major - breaking change
minor - added functionality (non-breaking)
patch - non logic changing changes, bugs, everything else

I was working under the assumption that patch was just for bug fixes. However, happy to fix! Just asking for clarification moving forward.